bitmovin/resources/models/inputs/gcs_input.py

```diff
@@ -19,8 +19,6 @@ def __init__(self, access_key, secret_key, bucket_name, cloud_region=None, id_=N
     def cloudRegion(self):
         if self._cloudRegion is not None:
             return self._cloudRegion
-        else:
```

Comment:
What's returned if it's None?
I think just `return self._cloudRegion` is sufficient without the if.